### ISSUE TITLE
Trying provided example

### ISSUE THREAD

**Issue by tikiram** [TechStark/express-mock-api-middleware#4]
Hi, I'm trying to execute your example but I'm getting this error, can you tell me what I'm doing wrong?

`_pathToRegexp` has several functions but `default`

```
$ node server.js
C:\Users\tikiram.ruiz\Repositories\api-with-mocks\node_modules\express-mock-api-middleware\lib\createMockMiddleware.js:168
      var re = (0, _pathToRegexp["default"])(path, keys, pathOptions); // debug(re);
                                            ^

TypeError: (0 , _pathToRegexp.default) is not a function
    at C:\Users\tikiram.ruiz\Repositories\api-with-mocks\node_modules\express-mock-api-middleware\lib\createMockMiddleware.js:168:45
    at Array.reduce (<anonymous>)
    at normalizeConfig (C:\Users\tikiram.ruiz\Repositories\api-with-mocks\node_modules\express-mock-api-middleware\lib\createMockMiddleware.js:152:32)
    at getConfig (C:\Users\tikiram.ruiz\Repositories\api-with-mocks\node_modules\express-mock-api-middleware\lib\createMockMiddleware.js:97:12)
    at getMockMiddleware (C:\Users\tikiram.ruiz\Repositories\api-with-mocks\node_modules\express-mock-api-middleware\lib\createMockMiddleware.js:48:18)
    at module.exports (C:\Users\tikiram.ruiz\Repositories\api-with-mocks\node_modules\express-mock-api-middleware\lib\index.js:8:47)
    at Object.<anonymous> (C:\Users\tikiram.ruiz\Repositories\api-with-mocks\server.js:5:65)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)

```

**Comment by tikiram** [comment]
Version `1.0.5` is working fine, version `1.0.8` is not working

**Comment by ttt43ttt** [comment]
Hi @Tikiram, thanks for your feedback. This is fixed now in `1.0.9`.
Also an example project is added (https://github.com/TechStark/express-mock-api-middleware/tree/develop/example).

**Comment by tikiram** [comment]
Thanks a lot
